jobs/JOBSPROC_DUMP_MONITOR

```diff
@@ -0,0 +1,277 @@
+export ushscript_dump=${ushscript_dump:-$USHbufr_dump}
+
+###############################################
+# Add some prod utilities to working directory
+###############################################
+echo "step ############# break ##############################" > ./break
+cp $UTILROOT/ush/err_chk   .; chmod +x err_chk
+cp $UTILROOT/ush/err_exit  .; chmod +x err_exit
+cp $UTILROOT/ush/postmsg   .; chmod +x postmsg
+cp $UTILROOT/ush/setpdy.sh .; chmod +x setpdy.sh
+
+##########################################
+# Run setpdy and initialize PDY variables
+##########################################
+set +x; echo -e "\n---> path to setpdy.sh below is: `which setpdy.sh`"; set -x
+setpdy.sh
+. $DATA/PDY
+
+#########################
+# Define COM directories
+#########################
+export COMIN_ROOT=${COMIN_ROOT:-${COMROOT:-""}}
+
+if [[ "$RUN_ENVIR" == nco ]]; then
+  export COMIN=${COMIN:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+  export COMOUT=${COMOUT:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+  mkdir -m 775 -p $COMOUT
+else
+  export COMIN=${COMIN:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+
+# COMOUT_ROOT for developers defaults to unique $DATA/com directory as root to
+#  prevent job from writing into a non-unique directory that might already have
+#  output in it from a previous checkout run for the same cycle
+# -----------------------------------------------------------------------------
+  export COMOUT_ROOT=${COMOUT_ROOT:-${DATA}/com}
+  export COMOUT=${COMOUT:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+  mkdir -m 755 -p $COMOUT
+fi
+
+# comin and comin_m1 are paths to dump status files for bufr_avgdata.sh
+export comin=${comin:?}/obsproc/${obsproc_ver}/${RUN}.${PDY}  #To be used in the future
+export comin_m1=${comin_m1:-${COMIN_ROOT:?}/obsproc/${obsproc_ver}/${RUN}.${PDYm1}}
+
+export comin=${comin:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+export comin_m1=${comin_m1:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/${RUN}.${PDY})}
+
+# ---------------------------
+# Paths for non-obsproc items
+# ---------------------------
+
+# TANK is root directory path to observational database used in dumpjb
+# --------------------------------------------------------------------
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/prod/dcom} 
+export TANK=${TANK:-${DCOMROOT}}
+
+# directory path to data count averaging executed in model script and in
+#  bufr_avgdata.sh
+# ----------------------------------------------------------------------
+export HOMEARCH=${HOMEarch:-$(compath.py ${envir}/${obsNET}/${obsproc_ver}/sdm_rtdm/$RUN)}
+
+#  .. AVGDarch_OUT is directory path for avg tables output by model script
+#     --------------------------------------------------------------------
+if [[ "$RUN_ENVIR" == nco ]]; then
+  export AVGDarch_OUT=$HOMEarch/avgdata  #### TALK TO NCO Steven Earle
```

Comment:
We got this resolved, so you can remove the inline comment:
`#### TALK TO NCO Steven Earle`